docs/developer/how-to.md

```diff
@@ -17,3 +17,77 @@ Open your browser at:
 - http://localhost:8080/swagger-ui/ for the old swagger 1.2 API (old swagger, endpoints will disappear from here (and become available in the previous link) as time progresses)
 
 Swagger json (for swagger 2.0 endpoints) is available under http://localhost:8080/api/swagger.json
+
+## How to run federation tests across two backends
+
+Requirements:
+
+* `helm` and `kubectl` available on your PATH
+* access to a kubernetes cluster via e.g. a `~/.kube/config` file.
+
+Using the latest code on the `develop` branch:
+
+First, find the latest CI-compiled code made available as docker images:
+
+```
+# Run all commands from the top directory of wire-server
+make latest-tag
+```
+
+Output might be
+
+```
+./hack/bin/find-latest-docker-tag.sh
+latest tag for brig:
+2.104.11
+latest tag for nginz:
+2.104.11
+```
+
+Let's assume the tags are the same(*) for both, then export an environment variable:
+
+```
+export DOCKER_TAG=2.104.11
+```
+
```

Comment:
Could this be automated?

Reply:
CI already runs these tests, so there it's already automated. For local purposes, chances are that you don't want to run these tests often, and if you do, you probably want control over which code should be tested. That makes further automating the "test the code on develop from a local computer" probably not very useful.